packages/astro/src/core/messages.ts

```diff
@@ -99,6 +99,8 @@ export function serverStart({
 		...localUrlMessages,
 		...networkUrlMessages,
 		'',
+		`${dim('Press')} ${bold('h + enter')} ${dim('to')} ${bold('show hot keys')}`,
+		''
```

Comment:
I don't think we want to show this message yet as proposed by Fred before, but an alternate tip display. ([Discord discussion](https://discord.com/channels/830184174198718474/1070208455329198190/1268119257263116401))

Reply:
Doesn't that just apply to persistent footers though? This is just logged at the same time the server URLs are. 
![image](https://github.com/user-attachments/assets/1b370452-6d16-4657-b77b-6ce5fa59015e)

Reply:
Yeah, it was also where I added the logs in the past, but we later discussed to not want to log it here for now.

Reply:
Aah. I will remove it from this PR (which is probably sensible anyway as it'll be experimental) and will make the case with Fred that content layer tips the balance in favour of adding it once it's stable!